src/main/java/com/prgrms/needit/common/error/ErrorCode.java

```diff
@@ -5,6 +5,7 @@
 @Getter
 public enum ErrorCode {
 	INTERNAL_SERVER_ERROR(500, "C001", "정의되지 않은 에러가 발생했습니다."),
+	CONTRACT_NOT_FOUND(404, "???", "해당 기부 예약을 찾을 수 없습니다."), // message source 적용?
 	INVALID_INPUT_VALUE(400, "C002", "올바른 입력 형식이 아닙니다.");
```

Comment:
기부 예약 관련 예외면 아래로 구분하여 따로 두면 될 것 같아요! 
위 C001, C002 는 공통으로 생길 수 있는 부분이고 기부 예약과 관련된 알파벳으로 정의해주세요 :)
```suggestion
        INTERNAL_SERVER_ERROR(500, "C001", "정의되지 않은 에러가 발생했습니다."),
        INVALID_INPUT_VALUE(400, "C002", "올바른 입력 형식이 아닙니다."),
        
	NOT_FOUND_CONTRACT(404, "???", "해당 기부 예약을 찾을 수 없습니다."); // message source 적용?
```

Reply:
원래는 Contract의 C로 하려고 했는데 이미 있어서.. 코드를 뭘로 할 지 고민 좀 해볼게요!